src/gallery_extra/bem_hardcoded/assemble_BEM.jl

```diff
@@ -0,0 +1,84 @@
+include("deHoop.jl");
+function assemble_BEM(lambda, mesh, gauss_order,der=0)
```

Comment:
Could the pre-computed code below that's not dependent on the lambda be stored / cached in `BEM_NEP`? I'm thinking so that they wouldn't have to be re-calculated each time `compute_Mder` is run. (But perhaps this is not a bottleneck anyway.)

Reply:
ThanksA It's not a bottleneck, but I did it anyway since it seems cleaner to also have precomputation for the assembly (not just the mesh generation).